extensions/amp-accordion/1.0/test/test-amp-accordion.js

```diff
@@ -184,7 +184,7 @@ describes.realWin(
       await waitForExpanded(sections[1], true);
 
       const getExpandedCount = () =>
-        element.querySelectorAll('[expanded]').length;
+        element.querySelectorAll('[aria-expanded="true"]').length;
       expect(getExpandedCount()).to.equal(2);
```

Comment:
nit: can be done without a method.
```
expect(element.querySelectorAll('aria-expanded="true')).to.have.lengthOf(2)
```

Reply:
The method is used again later in the `waitFor`, so I think it's a bit clearer to leave as is. Thanks for the rec.